packages/grid/test/filtering.test.js

```diff
@@ -206,8 +206,11 @@ describe('filtering', () => {
 
     grid._filters[0].value = '99';
     flushFilters(grid);
+
+    grid.allRowsVisible = true;
```

Comment:
Previously the test passed with the invalid value of 18 items only because the 19th item wasn’t actually rendered to the DOM (the grid wasn’t tall enough).

Switch to using all-rows-visible because it makes sure all the remaining items after filtering are available in the DOM.

Using all-rows-visible at the beginning isn’t relevant for the test and would end up creating 1000 row elements, which could be slow. That’s why I only do it at the very end before checking the filtered item count.